src/test/TestGhoCcipSteward.t.sol

```diff
@@ -252,6 +252,105 @@ contract TestGhoCcipSteward is TestGhoBase {
     );
   }
 
+  function testDisableRateLimitFromNonZeroToZero() public {
+    RateLimiter.TokenBucket memory outboundConfig = MockUpgradeableLockReleaseTokenPool(
+      GHO_TOKEN_POOL
+    ).getCurrentOutboundRateLimiterState(remoteChainSelector);
+    RateLimiter.TokenBucket memory inboundConfig = MockUpgradeableLockReleaseTokenPool(
+      GHO_TOKEN_POOL
+    ).getCurrentInboundRateLimiterState(remoteChainSelector);
+
+    assertTrue(outboundConfig.isEnabled);
+    assertGt(outboundConfig.capacity, 0);
+    assertGt(outboundConfig.rate, 0);
+
+    assertTrue(inboundConfig.isEnabled);
+    assertGt(inboundConfig.capacity, 0);
+    assertGt(inboundConfig.rate, 0);
+
+    RateLimiter.Config memory disableLimitConfig = RateLimiter.Config({
+      isEnabled: false,
+      capacity: 0,
+      rate: 0
+    });
+
+    // disable both inbound & outbound config
+    vm.prank(RISK_COUNCIL);
+    GHO_CCIP_STEWARD.updateRateLimit(
+      remoteChainSelector,
+      disableLimitConfig.isEnabled,
+      disableLimitConfig.capacity,
+      disableLimitConfig.rate,
+      disableLimitConfig.isEnabled,
+      disableLimitConfig.capacity,
+      disableLimitConfig.rate
+    );
+
+    outboundConfig = MockUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL)
+      .getCurrentOutboundRateLimiterState(remoteChainSelector);
+    inboundConfig = MockUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL)
+      .getCurrentInboundRateLimiterState(remoteChainSelector);
+
+    assertFalse(outboundConfig.isEnabled);
+    assertEq(outboundConfig.capacity, 0);
+    assertEq(outboundConfig.rate, 0);
+
+    assertFalse(inboundConfig.isEnabled);
+    assertEq(inboundConfig.capacity, 0);
+    assertEq(inboundConfig.rate, 0);
+  }
+
+  function testDisableRateLimitOnlyOutboundConfig() public {
```

Comment:
similar here, I d rename to `testChangeEnabledRateLimitOnlyOneSide`

Reply:
thanks, better naming! [a194c76](https://github.com/aave/gho-core/pull/427/commits/a194c762bba837301997472c2cbe873ce494b756)